src/test/java/org/jabref/gui/fieldeditors/LinkedFileViewModelTest.java

```diff
@@ -283,16 +284,16 @@ void isSamePath() {
     // Tests if added parameters to mimeType gets parsed to correct format.
     @Test
     void mimeTypeStringWithParameterIsReturnedAsWithoutParameter() {
-        Globals.prefs = JabRefPreferences.getInstance(); // required for task execution
-        ExternalFileTypes externalFileTypes = ExternalFileTypes.getInstance();
-        Optional<ExternalFileType> test = externalFileTypes.getExternalFileTypeByMimeType("text/html; charset=UTF-8");
+        Globals.prefs = jabRefPreferences.getInstance(); // required for task execution
+        ExternalFileTypes externalFileTypesInstance = externalFileType.getInstance();
```

Comment:
same as above and you don't need to call getInstance , the mock already creates an instance for you

Reply:
fixed.